tests/unit/test-cmd/test.run.js

```diff
@@ -325,5 +326,101 @@ describe('run', () => {
            sourceDir: '/fake/source/dir',
          },
        );
-     });
-});
+     }
+  );
+
+  describe('profile-create-new option', () => {
+    beforeEach(() => {
+      sinon.stub(fs, 'mkdir');
+    });
+
+    afterEach(() => {
+      fs.mkdir.restore();
+    });
+
+    it('creates dir when firefox profile doesn\'t exist',
+       async () => {
+         sinon.stub(fs, 'existsSync').returns(false);
+         const firefoxProfile = '/pretend/path/to/Firefox/profile';
+         const cmd = prepareRun();
+
+         await cmd.run({
+           firefoxProfile,
+           profileCreateIfMissing: true,
+         });
+
+         sinon.assert.calledWith(fs.mkdir, firefoxProfile);
+
+         fs.existsSync.restore();
+       }
+    );
+
+    it('creates dir when chromium profile doesn\'t exist',
```

Comment:
Nitpick similar to the previous one: `creates a chromium profile directory if it does not exist`